package.json

```diff
@@ -91,18 +91,19 @@
         "is-mobile": "^3.1.1"
     },
     "devDependencies": {
+        "@babel/core": "^7.0.0",
         "@changesets/changelog-github": "^0.4.3",
         "@changesets/cli": "^2.21.1",
         "@emotion/eslint-plugin": "^11.7.0",
-        "@emotion/react": "^11.1.5",
+        "@emotion/react": "11.1.5",
         "@guardian/ab-core": "^2.0.0",
-        "@guardian/commercial-core": "^4.25.0",
-        "@guardian/consent-management-platform": "^10",
-        "@guardian/eslint-plugin-source-foundations": "^4.0.2",
-        "@guardian/eslint-plugin-source-react-components": "^4.1.0",
-        "@guardian/libs": "^9.0.1",
-        "@guardian/source-foundations": "^4.0.2",
-        "@guardian/source-react-components": "^4.0.1",
+        "@guardian/commercial-core": "4.25.0",
```

Comment:
Are we intentionally fixing the versions or did we accidentally drop the `^`?

Reply:
@arelra fixed in https://github.com/guardian/atoms-rendering/pull/491